sops.go

```diff
@@ -100,7 +100,8 @@ type TreeItem struct {
 // TreeBranch is a branch inside sops's tree. It is a slice of TreeItems and is therefore ordered
 type TreeBranch []TreeItem
 
-// Trees usually have more than one branch
+// TreeBranches is a collection of TreeBranch
```

Comment:
This looks like it might all be one sentence, let's do:

```
TreeBranches is a collection of TreeBranch.
Trees usually have more than one branch.
```

To clarify

Reply:
✅